extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -94,9 +94,15 @@ describe('amp-a4a', () => {
         sandbox.spy(AmpA4A.prototype, 'onAmpCreativeRender');
     getSigningServiceNamesMock.returns(['google']);
     xhrMockJson.withArgs(
-        'https://cdn.ampproject.org/amp-ad-verifying-keyset.json',
-        {mode: 'cors', method: 'GET'})
-    .returns(Promise.resolve({keys: [JSON.parse(validCSSAmp.publicKey)]}));
+      'https://cdn.ampproject.org/amp-ad-verifying-keyset.json',
+      {
+        mode: 'cors',
+        method: 'GET',
+        requireAmpResponseSourceOrigin: false,
+        disableAmpSourceOrigin: true,
```

Comment:
`ampCors` is more than CORS. The extra things it does are the 2 I listed. And I don't see use case for:
```
requireAmpResponseSourceOrigin: true,
disableAmpSourceOrigin: true,
```

On the other side, `requireAmpResponseSourceOrigin` only exists for backward compatibility. I hope some day we can get rid of it. So any new changes should not really depend on it .